manim/mobject/types/vectorized_mobject.py

```diff
@@ -950,6 +950,29 @@ class VGroup(VMobject):
     This can be used to group multiple :class:`~.VMobject` instances together
     in order to scale, move, ... them together.
 
+    To add :class:`~.VMobject`s to a :class:`~.VGroup`, you can either use the
+    :meth:`~.VGroup.add` method, or straightforward addition. Similarly, you
+    can subtract elements of a VGroup via :meth:`~.VGroup.remove` method, or
+    subtraction operators:
+
+        >>> from manim import Triangle, Square
```

Comment:
maybe instead of this `from manim import *` ?

Reply:
I believe I've solved it, had a small typo on 972 with lower vs. uppercase s